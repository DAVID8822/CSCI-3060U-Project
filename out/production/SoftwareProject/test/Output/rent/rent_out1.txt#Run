Welcome to OT-BnB
Please enter option
create, delete, login, logout, post, search, exit
Please enter your username
Please enter your password
LOGIN SUCCESS! Welcome to OT-BnB testuser
Welcome to OT-BnB
Please enter option
create, delete, login, logout, post, search, exit
Please enter an id 
Please enter the number of nights you are staying
The rent per night is 120.32 and the total cost is 1203.1999999999998 Rent? YES/NO
Rental cancelled
Welcome to OT-BnB
Please enter option
create, delete, login, logout, post, search, exit
You have successfully logged out.
Welcome to OT-BnB
Please enter option
create, delete, login, logout, post, search, exit
