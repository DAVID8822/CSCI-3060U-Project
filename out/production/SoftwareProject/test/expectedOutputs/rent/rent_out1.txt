Welcome to OT-BnB
Please enter option
create, delete, login, logout, post, search, exit
Please enter your username
Please enter your password
LOGIN SUCCESS! Welcome to OT-BnB testuser
Welcome to OT-BnB
Please enter option
create, delete, login, logout, post, search, exit
Please enter an id 
Please enter the number of nights you are staying
The rent per night is 12.0 and the total cost is 120.0 Rent? YES/NO
Rental cancelled
Welcome to OT-BnB
Please enter option
create, delete, login, logout, post, search, exit
You have successfully logged out.
Welcome to OT-BnB
Please enter option
create, delete, login, logout, post, search, exit
